netplan/cli/commands/try_command.py

```diff
@@ -161,25 +160,16 @@ def is_revertable(self):  # pragma: nocover (requires user input)
         to not cleanly revert via the backend.
         '''
 
-        # Parse; including any new config file passed on the command-line:
-        # new config might include things we can't revert.
-        extra_config = []
-        if self.config_file:
-            extra_config.append(self.config_file)
-        self.config_manager.parse(extra_config=extra_config)
```

Comment:
How do we handle the `--config-file` passed as and CLI argument now?

Reply:
Good catch! Fixed.